how-to-guides/sagemaker/pytorch/Using-Neptune-with-PyTorch.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
     "# Using Neptune for logging PyTorch training jobs on SageMaker\n",
     "\n",
-    "<strong><font color='red'>This notebook should be run from the Amazon SageMaker Notebook.</font></strong>"
+    "<div class=\"alert alert-info\">This notebook should be run from the AWS SageMaker Notebook.</div>"
```

Comment:
These classes don't render in colab, so better to stick to HTML/markdown formatting